src/main/java/com/mecatran/gtfsvtor/dao/ReadOnlyDao.java

```diff
@@ -66,8 +66,9 @@
 
 	public Stream<GtfsTransfer> getTransfers();
 
-	public GtfsTransfer getTransfer(GtfsStop.Id fromStopId,
```

Comment:
Since the from/to route and trip ids are extended fields not part of the standard yet, maybe we could keep the getTransfer(fromstop, tostop) getter from the DAO alongside the new getter? (using null for other values as default).

Reply:
Yes, agreed. It's restored now.